WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -105,6 +107,24 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         NotificationCenter.default.addObserver(self, selector: #selector(showAddSelfHostedSite), name: .addSelfHosted, object: nil)
     }
 
+    private func showSitePicker(for blog: Blog) {
```

Comment:
There's an issue with displaying the site picker for the following flow:

1. Turn the My Site Dashboard flag to on
2. Log out
3. Login again and select a site
4. Tap "Show me around"
5. No site picker is shown

Reply:
Fixed! 7884be2